CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [1.7.0] - 2021-06-07
+- Migrate to AndroidX
```

Comment:
The changes for the 1.7.0 release need to be annotated with relevant headings in the CHANGELOG according to the keywords found here: http://keepachangelog.com/en/1.0.0/ (also see the entries for earlier releases, e.g. changes listed under headings like "Fixed", "Changed", "Removed" where appropriate).